packages/neon-core/src/rpc/RPCClient.ts

```diff
@@ -20,6 +20,48 @@ export interface GetUnclaimedResult {
   unclaimed: number;
 }
 
+export interface BlockHeader {

```

Comment:
Because these are using WitnessLike, they should be named with the -Like convention.